plugins/basic/nexus-repository-nuget/src/main/resources/com/sonatype/nexus/repository/nuget/internal/elasticsearch-mapping.json

```diff
@@ -18,21 +28,45 @@
           "properties": {
             "nuget": {
               "properties": {
-                "tags": {
-                  "type": "string",
-                  "index": "analyzed"
-                },
-                "summary": {
-                  "type": "string",
-                  "index": "analyzed"
-                },
-                "description": {
+                "title": {
                   "type": "string",
                   "index": "analyzed"
                 }
               }
             }
           }
+        },
+        "assets": {
+          "properties": {
+            "attributes": {
+              "properties": {
+                "nuget": {
+                  "properties": {
+                    "description": {
+                      "type": "string",
+                      "index": "analyzed"
+                    },
+                    "keywords": {
```

Comment:
Actually, the 'authors' field is included in the derived 'keywords' field, but this is really minor.